windows/installer.nsi

```diff
@@ -281,7 +288,11 @@ Section "un.Install"
     ${EndIf}
 
     ; Remove the Start Menu and desktop shortcuts
-    Delete   "$DESKTOP\${APP_NAME}.lnk"
+    IfFileExists "$DESKTOP\${APP_NAME}.lnk" deletedtshortcut continueremove
```

Comment:
The `deletedtshortcut` is visually hard to parse...  Perhaps just `deleteshortcut`.  Also, I'd prefer `continueremove` to be `skipshortcut`, as it's the "else" to this "if".

Reply:
Should be done now